src/runtime-rs/crates/hypervisor/src/device/driver/virtio_net.rs

```diff
@@ -28,25 +33,90 @@ impl fmt::Debug for Address {
     }
 }
 
-#[derive(Clone, Debug, Default)]
-pub struct NetworkConfig {
-    /// for detach, now it's default value 0.
-    pub index: u64,
+#[derive(Clone, Debug)]
+pub enum NetworkBackend {
+    Virtio(VirtioConfig),
+    Vhost(VirtioConfig),
+}
```

Comment:
What's the difference between the two backends? After read the related code, I find dup codes.

Reply:
The configs, used by virtio-net and vhost-net, are same. However, their backends are different, see `src/dragonball/src/api/v1/vmm_action.rs` line 305-318.

Reply:
Fixed, see 301551eee3719f43d6fccb6126575f43e6aef57e.